src/ui/Island/Cursor/CursorLabel.tsx

```diff
@@ -33,7 +43,7 @@ export default function CursorLabel({
           {name}
         </div>
         {realmIcon && (
-          <Icon src={realmIcon} type="image" height="16px" width="16px" />
+          <Icon src={realmIcon[1]} type="image" height="16px" width="16px" />
```

Comment:
I think we don't need `realmIcon`

```suggestion
          <Icon src={realmIcons[realmName?.toLowerCase()]} type="image" height="16px" width="16px" />
```